ui/tests/integration/components/confirmation-modal-test.js

```diff
@@ -0,0 +1,34 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import sinon from 'sinon';
+import { fillIn, find, render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | confirmation-modal', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it works as expected', async function(assert) {
```

Comment:
this test description makes me chuckle, but we should probably update it to describe what it's actually testing 😉

something a la `it renders with a disabled confirm button` or similar